terraform/modules/emr/security_groups.tf

```diff
@@ -55,3 +56,64 @@ resource "aws_security_group_rule" "ingress_https_from_emr_to_dks" {
   type              = "ingress"
   cidr_blocks       = toset(var.vpc.aws_subnets_private[*].cidr_block)
 }
+
+resource "aws_security_group_rule" "egress_emr_nodes" {
```

Comment:
You'll need a matching ingress rule for this to work.

Reply:
Corrected.